ledger/participant-integration-api/src/main/scala/platform/store/backend/StorageBackend.scala

```diff
@@ -413,3 +415,29 @@ trait StringInterningStorageBackend {
       connection: Connection
   ): Iterable[(Int, String)]
 }
+
+trait UserManagementStorageBackend {
+
+  def createUser(user: User, createdAt: Long)(connection: Connection): Int
+
+  def deleteUser(internalId: Int)(connection: Connection): Boolean
+
+  def getUser(id: UserId)(connection: Connection): Option[DbDto.User]
```

Comment:
@nmarton-da Not sure what's better:
a) returning a DbDto.User
b) or (Int, domain.User)
where that Int is user's internalId

Reply:
pls dont create a DbDto.User: that is an indexer side artifact. This is independent from those.
I would maybe create a case class instead of the pair, like DbUser(id: Int, user: domain.User)